### ISSUE TITLE
document with every theme-able quantity

### ISSUE THREAD

**Issue by Datseris** [JuliaPlots/Makie.jl#1304]
https://makie.juliaplots.org/dev/documentation/theming/#theming 

My difficulty is, I personally do not know a-priory how the thing I want to theme is actually named in the `set_theme` function. The page has some examples, but I think here it is good to be explicit. The most confusing thing is under which "subsection" of the theme do stuff go. Under `Axis = ...`, or in the general `set_theme()` function? For example, keys rowgap and colgap to change the default grid layout gaps. But where do I actually put those keys? `set_theme`, `Figure` or `Axis` subsection?

matplotlib has this sample file https://matplotlib.org/stable/tutorials/introductory/customizing.html#a-sample-matplotlibrc-file  which I always found extremely useful when theming my projects. Something like that would help increase transparency and reduce experimentation from the user side, which can be frustrating.

**Comment by Datseris** [comment]
E.g., explicitly writing the default `Theme` in the docs, with _every_ themeable thing and its default values, would be very helpful.

**Comment by Datseris** [comment]
Hm, by reading the page a couple of times more, I guess the approach is: "Use the uppercase name of the thing you want to theme, and use the keywords the thing anyways takes as the keys." That's kinda clear. if this default `Theme` already exists somewhere written, I think it would still be helpful to put there, but if it takes effort to write it I guess we can close this. 

Perhaps at the very opening of the doc page, it is worth stating this approach to theming: "Use the uppercase name of the thing you want to theme, and use the keywords the thing anyways takes as the keys."

**Comment by jkrumbiegel** [comment]
> Use the uppercase name of the thing you want to theme, and use the keywords the thing anyways takes as the keys

Yes that was exactly my idea, but I agree it would be useful to list it explicitly.

> keys rowgap and colgap to change the default grid layout gaps

The problem with these is that GridLayout is not a Makie type, so it's tricky to define good default behaviors. The default getter for this value is actually quite ugly.. https://github.com/jkrumbiegel/GridLayoutBase.jl/blob/master/src/GridLayoutBase.jl#L11-L12

**Comment by ffreyer** [comment]
Can this be closed?
